tests/test_bootstrap.py

```diff
@@ -371,7 +371,7 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     assert "an_after_dep" not in hass.config.components
     assert "an_after_dep_of_after_dep" not in hass.config.components
     assert "an_after_dep_of_after_dep_of_after_dep" not in hass.config.components
-    assert order == ["cloud", "normal_integration"]
+    assert order == ["normal_integration", "cloud"]
```

Comment:
```suggestion
    # normal_integration is an after dependency of cloud
    assert order == ["normal_integration", "cloud"]
```

Reply:
On second thought, I think it's clear from the test why the order is the way it is